lib/ui/channel_buffers.dart

```diff
@@ -150,14 +149,14 @@ class ChannelBuffers {
   }
 
   /// Returns null on underflow.
-  _StoredMessage _pop(String channel) {
-    final _RingBuffer<_StoredMessage> queue = _messages[channel];
-    final _StoredMessage result = queue?.pop();
+  _StoredMessage? _pop(String channel) {
+    final _RingBuffer<_StoredMessage>? queue = _messages[channel];
+    final _StoredMessage? result = queue?.pop();
     return result;
   }
 
   bool _isEmpty(String channel) {
-    final _RingBuffer<_StoredMessage> queue = _messages[channel];
+    final _RingBuffer<_StoredMessage>? queue = _messages[channel];
     return (queue == null) ? true : queue.isEmpty;
```

Comment:
Can be written as `return queue == null || queue.isEmpty;`

(You never need a `?`/`:` operator where one of the branches is a literal boolean)

Reply:
Done.